main.cpp

```diff
@@ -4923,6 +4923,12 @@ void sign_guts_elf(elf_file* elf, private_t private_key, public_t public_key) {
         image_type->flags |= PICOBIN_IMAGE_TYPE_EXE_TBYB_BITS;
     }
 
+    if (settings.seal.set_tbyb) {
```

Comment:
note this was kicking around in the branch - @will-v-pi not sure if it belongs or not